Dfe.Academies.Academisation.Core/CreateResult.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Dfe.Academies.Academisation.Core;
 
-public abstract class CreateResult<TPayload> : CommandOrCreateResult
+public abstract class CreateResult : CommandOrCreateResult
```

Comment:
Why has this type been removed?

Reply:
It's not used anywhere - internally within the type, or elsewhere in the codebase.